src/server/services/user.service.ts

```diff
@@ -356,6 +356,23 @@ export const toggleBlockedTag = async ({
   return dbWrite.tagEngagement.create({ data: { userId, tagId, type: 'Hide' } });
 };
 
+export const updateAccountScope = async ({
+  userId,
+  provider,
+  scope,
+}: {
+  userId: number;
+  provider: string;
+  scope: string;
+}) => {
+  const account = await dbWrite.account.findFirst({
+    where: { userId, provider },
+    select: { id: true, scope: true },
+  });
+  if (account && account.scope != scope)
```

Comment:
Just nitpicking
```suggestion
  if (account && account.scope !== scope)
```

Reply:
I took it a step further and adjusted the comparison to be an actual comparison of the values in the scope, not just the string. This will help avoid updating scope if they're just in a different order.